raiden_contracts/deploy/__main__.py

```diff
@@ -127,6 +127,16 @@ def deploy(
         )
         return receipt
 
+    def transact(
+        self,
+        to_be_called,
```

Comment:
Let's add types to all new production code (we can be less strict with tests)! In this case `Callable`.

Reply:
OK.  Will do.

Reply:
Turns out `mypy` caught my misconception.